deploy/examples/cluster-on-local-pvc-test.yaml

```diff
@@ -0,0 +1,174 @@
+#################################################################################################################
+# Define the settings for the rook-ceph cluster with common settings for a production cluster on top of bare metal.
+
+# This example expects three nodes, each with two available disks. Please modify it according to your environment.
+# See the documentation for more details on storage settings available.
+
+# For example, to create the cluster:
+#   kubectl create -f crds.yaml -f common.yaml -f operator.yaml
+#   kubectl create -f cluster-on-local-pvc.yaml
```

Comment:
`cluster-on-local-pvc.yaml` does not match the file name `cluster-on-local-pvc-test.yaml`